parsing.py

```diff
@@ -153,10 +153,12 @@ def url_to_shortlink(url):
         return url
     if id_and_site[2] == "question":
         return "http://{}/questions/{}".format(id_and_site[1], id_and_site[0])
+
```

Comment:
Oh, we're referring to two different places. I'm taking about above & below the block comment, that whitespace. The marker must have shifted with the changes (or I placed it wrong...)